internal/testutil/pkgdata.go

```diff
@@ -274,3 +274,35 @@ func MustMakeDeb(entries []TarEntry) []byte {
 	}
 	return data
 }
+
+func REG(mode int64, path, content string) TarEntry {
```

Comment:
nit: These all caps shorthand versions does not feel right quite to me.

I do not think these fall under [initialisms or recognised abbreviations](https://github.com/golang/go/wiki/CodeReviewComments#initialisms), which should be all caps. Instead I think these closer match some of the shorthand forms as found in, for example, the https://pkg.go.dev/syscall package.

I would take some inspiration from these examples:

```Dup()``` https://pkg.go.dev/syscall#Dup
```Acct()``` https://pkg.go.dev/syscall#Acct

Reply:
Maybe even ```File()```, ```Dir()``` and ```Link()``` would read nicely.

Reply:
Can we reach a compromise and use `Reg()`, `Dir()` and `Lnk()` from tar nomenclature? I understand the desire for readable names, but per my commit message:
```
The reason for the 3 letter names and the order of arguments is to make
the list of paths aligned on the same column in tarball definitions.
```
in this case I'd much rather have readable lists of paths. In followup commit there's going to be a lot of such lists in test cases. Once the reader grasps the meaning of the function names, it's much easier to follow the test cases written as such
```
Dir(0755, "./data/"),
Reg(0600, "./data/document.txt", "words words words"),
Lnk(0777, "./data/document", "document.txt"),
Reg(0777, "./data/ebook1", "tale"),
Reg(0777, "./data/ebook2", "another tale"),
Reg(0777, "./data/ebook3", "manual"),
Dir(0755, "./data/pdfs/"),
Reg(0777, "./data/pdfs/lectures1.pdf", "physics"),
Reg(0777, "./data/pdfs/lectures2.pdf", "chemistry"),
Reg(0777, "./data/pdfs/lectures3.pdf", "history"),
```
instead of misaligned
```
Dir(0755, "./data/"),
File(0600, "./data/document.txt", "words words words"),
Lnk(0777, "./data/document", "document.txt"),
File(0777, "./data/ebook1", "tale"),
File(0777, "./data/ebook2", "another tale"),
File(0777, "./data/ebook3", "manual"),
Dir(0755, "./data/pdfs/"),
File(0777, "./data/pdfs/lectures1.pdf", "physics"),
File(0777, "./data/pdfs/lectures2.pdf", "chemistry"),
File(0777, "./data/pdfs/lectures3.pdf", "history"),
```
I've pushed the change to camel cased names. If you are fine with this compromise, please resolve the thread.